functions/supplier/debezium-supplier/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.cloud.stream.kafka.default.producer.messageKeyExpression=headers['debezium_key'].bytes
```

Comment:
This one has to go away from the `supplier` module.

Reply:
sure